DataFormats/L1TGlobal/interface/GlobalAlgBlk.h

```diff
@@ -92,12 +90,6 @@ class GlobalAlgBlk
 
 private:
 
-    /// orbit number
-    int m_orbitNr;
-
-    /// bunch cross number of the actual bx
-    int m_bxNr;
-
```

Comment:
uGT read-out block does not [0] and will not [1] contain orbit/bx number.
[0] https://twiki.cern.ch/twiki/bin/viewauth/CMS/GlobalTriggerUpgrade-ReadoutSpec
[1] https://twiki.cern.ch/twiki/bin/viewauth/CMS/UGT_readout_proposal